<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnaPencere</class>
 <widget class="QMainWindow" name="AnaPencere">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Telefon Defteri</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>511</width>
      <height>421</height>
     </rect>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>481</width>
        <height>411</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Kayıt ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lblID">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Adı:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="txtAdi"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Soyadı:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="txtSoyadi"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Telefon</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="txtTelefon"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>E Posta</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="txtPosta"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>İLİ</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="cmbIl"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>İLÇESİ</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="cmbIlce"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>  Liste</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QTableWidget" name="tblListe"/>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="btKaydet">
         <property name="text">
          <string>Kaydet</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QPushButton" name="btVazgec">
         <property name="text">
          <string>Vazgeç</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>524</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="kayit">
    <property name="title">
     <string>Kayıt</string>
    </property>
    <widget class="QMenu" name="telefon">
     <property name="title">
      <string>Telefon Listesi</string>
     </property>
     <addaction name="TelEkleme"/>
     <addaction name="TelListeleme"/>
    </widget>
    <addaction name="telefon"/>
    <addaction name="actionEposta_Listesi"/>
   </widget>
   <addaction name="kayit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEposta_Listesi">
   <property name="text">
    <string>Eposta Listesi</string>
   </property>
  </action>
  <action name="TelEkleme">
   <property name="text">
    <string>Ekleme</string>
   </property>
  </action>
  <action name="TelListeleme">
   <property name="text">
    <string>Listeleme</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
